--- docs/conf.py	2015-09-27 08:54:46.000000000 +0200
+++ docs/conf.py	2015-10-17 18:43:07.013014490 +0200
@@ -17,6 +17,7 @@
 import os
 import shlex
 import time
+import alabaster
 
 try:
     import sphinx_rtd_theme
@@ -174,7 +175,7 @@
 #html_theme_options = {}
 
 # Add any paths that contain custom themes here, relative to this directory.
-#html_theme_path = []
+html_theme_path = [alabaster.get_path()]
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
